src/test/unit/test_lumigo_utils.py

```diff
@@ -467,24 +473,16 @@ def test_report_json_extension_spans_mode(monkeypatch, reporter_mock):
         )
     report_json(region=None, msgs=spans, is_start_span=False)
 
-    files_paths = []
-    for span in spans:
-        files_paths.append(get_span_file_name(span, "span"))
-    done_object = {"spansCount": len(spans) + 1}
-    files_paths.append(get_span_file_name(done_object, "done"))
-    files = glob.glob("/tmp/lumigo-spans/*")
-    assert len(files) == size_factor + 1
-    for file_name in files:
-        file_content = open(file_name, "r").read()
-        suffix_len = len(EXTENSION_FILE_SUFFIX)
-        suffix = file_content[-suffix_len:]
-        json.loads(file_content[:-suffix_len])
-        assert suffix == EXTENSION_FILE_SUFFIX
-        assert file_name in files_paths
-
-
-def get_span_file_name(span, _type):
-    to_send = aws_dump(span).encode() + EXTENSION_FILE_SUFFIX.encode()
+    start_file_path = get_span_file_name(start_span, "span")
+    end_file_path = get_span_file_name(spans, "end")
+    start_file_content = json.loads(open(start_file_path, "r").read())
+    end_file_content = json.loads(open(end_file_path, "r").read())
+    assert start_span == start_file_content
+    assert json.dumps(end_file_content) == json.dumps(spans)
```

Comment:
why do you compare strings here?
you can just assert `end_file_content == spans`, right?

Reply:
one is [{'0': 'a'}] and the other is [{0: 'a'}] so i dump it so its equal